gokart/redis_lock.py

```diff
@@ -80,6 +92,70 @@ def extend_lock():
     return wrapper
 
 
+def wrap_with_run_lock(func, redis_params: RedisParams):
+    return _wrap_with_lock(func=func, redis_params=redis_params)
+
+
+def wrap_with_dump_lock(func: Callable, redis_params: RedisParams, exist_check: Callable):
+    # When TargetOnKart.dump() is called, dump() must be wrapped with redis lock and cache existance check.
+    # https://github.com/m3dev/gokart/issues/265
```

Comment:
This comment and detailed description of this feature should documented on `docstring` and gokart doc.

Reply:
@hirosassa Thank you for the comment! I've moved these comments to the docstring.

These methods are only used internally and not to be used by users. So, I think it should not be included in the documents to make the documents simpler. How do you think of this?

Reply:
@mski-iksm 
> I think it should not be included in the documents

OK! I understood 👍